README.md

```diff
@@ -538,6 +538,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Dhaval Dudhat](https://github.com/dudhatdhavalm)
   - [Dhayal Ram](https://github.com/dhayalramk)
   - [Dheeraj Khushalani](https://github.com/DheerajKhush)
+  - [Dheeman Pati](https://github.com/sayan99614)
```

Comment:
Hey @sayan99614, Looks like you didn't add your name in alphabetical order! Please add your name before `Dheeraj Khushalani`.

Please let us know if you need any help!

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Inefficient

Reply:
changed the sequence!